### PR TITLE
add: basic rr 0.5.6 questions

### PR DESCRIPTION
**Pull request by RuckusRoboticsCode**

### CODE REVIEW

**Comment by 0verkil** [comment]
can we add prefix pages on subsections? seems like LRR would be a useful source to link here
also, we should get a style guide set up lol
otherwise, stuff looks good to me, though i'm not sure if this accounts for all 0.5.x and 1.0.x stuff since i'm not familiar enough with rr

File: src/Roadrunner 0.5.6/manual_feedforward_tuner_overshooots.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Manual Feedforward Tuner Overshoots
+
+This is normal! The REV hub's motor controller are not great at decelerating, so this typically causes about a 10% overshoot on manual feedforward tuner. It's okay to move on the the next tuning steps. However, when you get to the feedback tuning, whether it's "back and forth" or "followerPIDTuner", you will want to add a non-zero kd term. This will help the robot not overshoot.
```
Comment (dr-hextanium) [inline src/Roadrunner 0.5.6/manual_feedforward_tuner_overshooots.md:3]: For RR 1.0.x, the feedback tuner is now called `ManualFeedbackTuner`.
In the future, when we have more on PID tuning, we should reference that page.

File: src/Roadrunner 0.5.6/robot_drifts_to_one_side_during_manual_feedforward_tuning.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Robot Drifts to One Side During Manual Feedforward Tuning
+
+If this happens, you shouldn't worry. This can be caused by an unbalanced robot or one wheel having slightly more friction than the others. Whatever the reason, this will be corrected for in the later tuning steps so you can safely ignore this drift.
```
Comment (dr-hextanium) [inline src/Roadrunner 0.5.6/robot_drifts_to_one_side_during_manual_feedforward_tuning.md:3]: 👍🏽

File: src/Roadrunner 0.5.6/manual_feedforward_tuner_overshooots.md (hunk 1 above)
Comment (dr-hextanium) [reply src/Roadrunner 0.5.6/manual_feedforward_tuner_overshooots.md:3]: I just realized that this is for 0.5.6 and that `BackAndForth` is apart of that 💀 
Ignore the above.

File: src/Roadrunner 0.5.6/robot_drives_full_speed_on_start_when_following_trajectory.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Robot Drive Full Speed on Start When Following Trajectory
+
+If you are running an op mode that has roadrunner trajectories in it, and when you start moving it goes at full speed right away, this almost always means you forgot to set a pose estimate. In `init()` before you run any trajectories, make sure you have `sampleMecanumDrive.setPoseEstimate(startingPose)`, whatever your starting pose may be. Also make sure that the trajectory starting pose matches this. Like
+```
+Trajectory traj = sampleMecanumDrive.trajectoryBuilder(startingPose)
```
Comment (dr-hextanium) [inline src/Roadrunner 0.5.6/robot_drives_full_speed_on_start_when_following_trajectory.md:5]: 👍🏽

File: src/Roadrunner 0.5.6/robot_makes_circle_when_spinning_in_place.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# Robot Makes Circle When Spinning In Place
```
Comment (dr-hextanium) [inline src/Roadrunner 0.5.6/robot_makes_circle_when_spinning_in_place.md:1]: I think this should be clarified further. Are you referring to the drift the robot experiences while rotating in place?

File: src/Roadrunner 0.5.6/robot_velocity_plateaus_below_target_velocity_plateau.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# Robot Velocity Plateaus Below Target Velocity Plateaus
+
+Okay, so this means you've reached your robot's actual max velocity. You should lower the max velocity specified in DriveConstants. Run the MaxVelocityTuner to find the recommended max velocity to use.
```
Comment (dr-hextanium) [inline src/Roadrunner 0.5.6/robot_velocity_plateaus_below_target_velocity_plateau.md:3]: 👍🏽

File: src/SUMMARY.md [hunk 6]
```diff
@@ -3,5 +3,21 @@
 [Introduction](./introduction.md)
 ---
 
+RoadRunner 0.5.6
+---
```
Comment (dr-hextanium) [inline src/SUMMARY.md:7]: Remove the separator here; you want to have this be a section title.

**Review by dr-hextanium — CHANGES_REQUESTED** [review]
Just a few minor changes before we can push! Overall, great job, and thank you for contributing :)

**Pull request closed without merging**